### ISSUE TITLE
Downloading the Detailed Report Should Include Raw Mark 

### ISSUE THREAD

**Issue by jessebett** [MarkUsProject/Markus#3368]
When I download the Detailed CSV Report for an assignment I would like there to be a column in the CSV that corresponds to the total marks the student received before any late penalties were applied.

Special cases sometimes require late penalties to be removed of reduced, and as it is right now this cannot be done in the MarkUs web interface without completely removing the TA's grades.

Instead, I would like to do this manually. Currently the CSV has a column for the late penalty that was applied. Please include a column totaling the marks before that penalty. This allows instructors to manually calculate percent score and apply late penalties if we need.

Thanks

**Comment by mishaschwartz** [comment]
@jessebett . You mention:

> Special cases sometimes require late penalties to be removed of reduced, and as it is right now this cannot be done in the MarkUs web interface without completely removing the TA's grades.

There is in fact a way to add, remove, or reduce penalties for individual students. 

When marking a student's assignment, if you navigate to the "Summary" tab (beside the "Marks" tab where you would fill in the criteria) there is a section called "Bonus/Deductions" where you can remove penalties, and add extra marks. These extra marks can be negative, which lets them act as a penalty. 

In the screen shot below you can see an example where a late penalty is added (note the "Remove" button) and an additional extra mark has been added as well.

![screen shot 2018-05-10 at 9 59 49 am](https://user-images.githubusercontent.com/4380924/39873437-f14122a2-5438-11e8-9014-541e75f4fed4.png)

**Comment by david-yz-liu** [comment]
@mishaschwartz you're right, but I think there's still room for improvement on what's actually downloaded. I'll leave this open for now.
